build/install.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+set -x
+
+# All of these commands are executed by the jenkins user.
+
+sudo apt-get install -y curl gnupg2 wget git
+
+# Docslave specifics
+# These packages are needed as a dependency of the nokogiri ruby gem (which in turn is a dependency of the sanitize ruby gem).
+
+sudo apt-get install -y libxslt-dev libxml2-dev
+
+# Python is needed for our pre and post processing scripts.
+
+sudo apt-get install -y python
+
+# These packages are needed to satisfy rvm requirements.
+
+sudo apt-get install -y libyaml-dev libsqlite3-dev sqlite3 autoconf libgmp-dev libgdbm-dev libncurses5-dev automake libtool bison pkg-config libffi-dev
+
+# We currently use pygments for syntax highlighting. Jekyll 0.12.1 errors during install without this.
+
+sudo apt-get install -y python-pygments
+
+# yui-compressor (ruby gem dependency) is written in java, so we need to install it.
+
+sudo apt-get install -y default-jdk
+
+sudo apt-get install -y libssl-dev zlib1g-dev
+
+# For making cfenigne
+# sudo apt-get -qy install bison flex binutils build-essential fakeroot ntp dpkg-dev libpam0g-dev liblmdb-dev libpcre3-dev
+
+#begin_make_nova
+# For making nova
+# sudo apt-get -qy install libpq-dev libpqtypes-dev libpgtypes3 libecpg-dev php7.0 php7.0-dev
+#end_make_nova
+
+
+# Install WeasyPrint for creating PDFs
+# sudo apt-get -y install python3-pip libcairo2-dev pango1.0-tests #also requires gcc installed earlier
+# sudo pip3 install weasyprint
+# weasyprint --help
+# will it work on second try?
+# sudo pip3 install weasyprint
+# weasyprint --help
+
+# install wkhtmltopdf for creating PDFs
+# sudo apt-get -y install xfonts-base xfonts-75dpi fonts-liberation
+# wget https://downloads.wkhtmltopdf.org/0.12/0.12.5/wkhtmltox_0.12.5-1.bionic_amd64.deb
+# sudo dpkg -i wkhtmltox_0.12.5-1.bionic_amd64.deb
```

Comment:
Remove the above commented-out lines or put them into some `========` section with an explanation of why we want to keep them here.